src/algorithms/all_shortest_paths.c

```diff
@@ -74,7 +83,10 @@ int AllShortestPaths_FindMinimumLength
 		if(depth < array_len(ctx->levels)) array_clear(ctx->levels[depth]);
 	}
 
+	GrB_free(&current_visited);
 	ctx->visited = visited;
+	GrB_Index nvals;
+	GrB_Vector_nvals(&nvals, visited);
```

Comment:
Why are we flushing `visited`?

Reply:
sorry needed this for debug to see that always we get same visited ovals